### PR TITLE
added rto

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference

[View RTO Reasons](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#12-view-rto-risk-reasons)
[Update fulfilment details](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#13-update-the-fulfillment-details)

### CODE REVIEW

File: documents/order.md [hunk 1]
```diff
@@ -275,6 +275,75 @@ Razorpay::Order.edit(orderId,para_attr)
...
+
+```go
+orderId = "order_XXXXXXXXXXXXX1"
+
+Razorpay::Order.fetch(orderId).view_rto
+```
+**Parameters**
+
+| Name     | Type   | Description                         |
+|----------|--------|-------------------------------------|
```
Comment (bhavyay) [inline documents/order.md:288]: please format the table

File: documents/order.md [hunk 2]
```diff
@@ -275,6 +275,75 @@ Razorpay::Order.edit(orderId,para_attr)
...
+    "provider": "Bluedart"
+  }
+}
+
+Razorpay::Order.fetch("order_XXXXXXXXXXXXX1").edit_fulfillment(para_attr)
+```
+**Parameters**
+
+| Name     | Type   | Description                         |
+|----------|--------|-------------------------------------|
```
Comment (bhavyay) [inline documents/order.md:328]: please format the table

File: test/razorpay/test_order.rb [hunk 3]
```diff
@@ -66,6 +66,27 @@ def test_fetch_order_transfers
...
+    order = Razorpay::Order.fetch(@order_id).view_rto
+    assert !order.rto_reasons.empty?, 'orders should be more than one'
+   end
+
+   def test_fulfillment
+      param_attr = {
+        "payment_method": "upi",
+        "shipping": {
+          "waybill": "123456789",
+          "status": "rto",
```
Comment (bhavyay) [inline test/razorpay/test_order.rb:82]: please add test case for failure scenario

Comment (ankitdas13) [reply test/razorpay/test_order.rb:92]: Added 

https://github.com/razorpay/razorpay-ruby/blob/1fcd3fa00ae1048e83a282d329e3f3d0bdf6b1cb/test/razorpay/test_order.rb#L102

File: documents/order.md (hunk 2 above)
Comment (ankitdas13) [reply documents/order.md]: formatted the table 

https://github.com/razorpay/razorpay-ruby/blob/1fcd3fa00ae1048e83a282d329e3f3d0bdf6b1cb/documents/order.md?plain=1#L327

File: documents/order.md (hunk 1 above)
Comment (ankitdas13) [reply documents/order.md]: formatted

https://github.com/razorpay/razorpay-ruby/blob/1fcd3fa00ae1048e83a282d329e3f3d0bdf6b1cb/documents/order.md?plain=1#L287

File: documents/order.md [hunk 4]
```diff
@@ -275,6 +275,75 @@ Razorpay::Order.edit(orderId,para_attr)
 -------------------------------------------------------------------------------------------------------
 
 
+### View RTO/Risk Reasons
+
+```go
+orderId = "order_XXXXXXXXXXXXX1"
+
+Razorpay::Order.fetch(orderId).view_rto
```
Comment (bhavyay) [inline documents/order.md:283]: please remove the method chaining.

File: documents/order.md [hunk 5]
```diff
@@ -275,6 +275,75 @@ Razorpay::Order.edit(orderId,para_attr)
...
+para_attr = {
+  "payment_method": "upi",
+  "shipping": {
+    "waybill": "123456789",
+    "status": "rto",
+    "provider": "Bluedart"
+  }
+}
+
+Razorpay::Order.fetch("order_XXXXXXXXXXXXX1").edit_fulfillment(para_attr)
```
Comment (bhavyay) [inline documents/order.md:323]: please remove the method chaining.

File: documents/order.md (hunk 4 above)
Comment (ankitdas13) [reply documents/order.md]: Updated doc

https://github.com/razorpay/razorpay-ruby/blob/c06d03c69ec3eeb816196e5a0ce5b663929c84f5/documents/order.md?plain=1#L283

**Review by bhavyay — APPROVED** [review]

**Pull request merged**